docs/git-comparison.md

```diff
@@ -127,13 +127,24 @@ parent.
     <tr>
       <td>Show diff of the current change</td>
       <td><code>jj diff</code></td>
-      <td><code>git diff HEAD</code></td>
+      <td><code>git diff</code></td>
```

Comment:
This only works if the staging area is empty. I think you should either revert this change or add `(assuming empty staging area)`.

Reply:
Ah, I didn't know that! Done.